src/components/CrisisLineItem.js

```diff
@@ -38,7 +38,7 @@ class CrisisLineItem extends React.Component {
                             <Link
                                 to={`/service/${object.slug}`}
                             >
-                                {object.site.name} {object.name}
+                                {object.site.name}
```

Comment:
The problem is you just get a name but it tells you nothing about what you'll get when you call. I.e. is it an IAP, or an infoline, etc. What would be better maybe is to apply some substitutions, like `/IAP/` to "assessment line" etc.